.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
     - master
   pull_request:
+  schedule:
+    - cron:  '1 0 * * *'
```

Comment:
What's the benefit to running the standard ci every day in addition to the beleeding-edge?

Reply:
The bleeding edge workflow didn't detect the formatting changes from black, so it's clearly not covering everything that normal CI does. My reasoning is that it's easier to just run both daily than trying to cover everything with bleeding edge.